### ISSUE TITLE
strelaypoolsrv: Google maps issue in the GUI

### ISSUE THREAD

**Issue by calmh** [syncthing/syncthing#5296]
We're not using the API key properly or something;

<img width="1814" alt="screen shot 2018-10-30 at 05 30 34" src="https://user-images.githubusercontent.com/125426/47696000-0284b980-dc05-11e8-8472-3f8c4b7661dd.png">

**Comment by AudriusButkevicius** [comment]
It looks like you need to pay now?

**Comment by calmh** [comment]
Or at least have an account with a credit card on file tied to the API key. This seems slightly onerous under the circumstances... :/

**Comment by imsodin** [comment]
That change wrecked quite a few softwares using it (e.g. https://discuss.pixls.us/t/google-maps-and-digikam/9547/3). Some opensource map api like leaflet or others would probably be the best option. but that's probably just an annoying amount of work...

**Comment by uok** [comment]
I spent quite a lot of time with this change :disappointed:  

In early days you did not need an API key or registration to use Google Maps. Then Google decided to make an API key mandatory (I guess a few heavy users causing this impulse or Google wants more money to satisfy stock holders?), but they could not inform the users about this change without any contact info. :man_facepalming: Result: all maps without key broken

A few months after everbody had updated their maps, Google decided to make payment details mandatory too - but they failed to clearly communicate this change. So all maps broken again :angry: 

I still see lots of (contact) pages with broken maps because you hardly visit your own page(s) (besides start page e.g. after wordpress update). Also average Joe is not your typical dev "I just register there and setup an API key" type.

:information_source: What is needed:
- Account at https://console.cloud.google.com (very confusing, because this is Google cloud not just Google maps)
- a credit card (or if you register as a company a bank account)

Then I found out that you get 200$ free monthly credit which covers 99% of all use cases. Also you can limit requests to your domain and set usage limits so you never spend more even with high traffic pages. With that perspective this whole API key thing is just plain stupid.

A few days ago they [changed](https://developers.google.com/maps/2016-customer-faq) things again (wtf?) and it looks like the free part is also disappearing over the next months (not clear if for all or just heavy users or old users or ...?) and I'm still not sure how much the average Joe that has a simple map on the contact page has to pay (or not). This whole thing is utterly confusing  - just take a look at the [pricing](https://cloud.google.com/maps-platform/pricing/sheet/) (scroll down to see what I mean)

If you find out more how this works out financially please let me know, thanks!

**Comment by uok** [comment]
@calmh did you figure out how much (or not) it costs for a few map loads?

**Comment by calmh** [comment]
I'm guessing $0 for our usage, we'll see I guess...

**Comment by AudriusButkevicius** [comment]
Data.syncthing.net needs love too

**Comment by calmh** [comment]
Yeah I'll add the same API key there somewhere, just haven't gotten around to it. Or you could. The API key is valid for use on https://*.syncthing.net/*

**Comment by calmh** [comment]
e

**Comment by AudriusButkevicius** [comment]
As I've mentioned elsewhere, I currently live out of a suitcase without access to my desktop. My laptop is just chrome on a bigger screen than my mobile, with no ssh keys or anything on it.
